app/Http/Middleware/CheckLocale.php

```diff
@@ -18,22 +19,24 @@ class CheckLocale
      */
     public function handle($request, Closure $next, $guard = null)
     {
+
+        // Default app settings from config
+        $language = config('app.locale');
+
         if ($settings = Setting::getSettings()) {
+
             // User's preference
             if (($request->user()) && ($request->user()->locale)) {
```

Comment:
Would we want to "respect the User's locale" even if there *is* no settings record? I don't know if that's a scenario we need to even think about.

Reply:
If there is not settings record, we have way bigger problems. The system won't work without it.